docs/anyedge/anyedge.md

```diff
@@ -51,39 +51,39 @@ When a packet traverses NAT, the UDP packet headers are correctly rewritten by y
 
 **4.** Far End NAT Traversal.
 
-We can use any of the following indicators to detect if NAT is present in the UAC.
+You can use any of the following indicators to detect if NAT is present in the UAC.
 
 + Search the Contact header  field for occurrences of RFC1918 / RFC6598 addresses.
 + Use the "received" test: "address in Via" to compare against the source IP address used for signaling.
 + Search the top-most VIA for the occurrences of RFC1918 / RFC6598 addresses.
 + Search the Session Description Protocol(SDP) for the occurrences of RFC1918 / RFC6598 addresses.
-+ Test if the source port is different from the "port in Via".
++ Test if the source port is different from the "port in Via."
 + Compare the "address in Contact" with the source IP address used for signaling.
 + Compare the "Port in Contact" with the source port used for signaling.
 
 ### Inbound Proxy / Dispatcher / Load Balancer
 
-The primary use case for **AnyEdge** is to disseminate calls to a pool of SIP Servers. These can be configured using the following call strategies:
+The primary use case for **AnyEdge** is to disseminate calls to a pool of SIP Servers. Configure the servers using the following call strategies:
```

Comment:
This does not read well.

Reply:
Rewritten as : 
The primary use case for *AnyEdge* is to distribute calls to the SIP Servers. You can use the following algorithms to distribute the calls: